sambaplayersdk/src/main/java/com/sambatech/player/cast/CastPlayer.java

```diff
@@ -130,11 +130,14 @@
   private int pendingSeekWindowIndex;
   private long pendingSeekPositionMs;
   private boolean waitingForInitialTimeline;
+  private boolean isLive;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "isLive" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.sambatech.com/coding_rules#rule_key=squid%3AS1068)